tests/nnstreamer_plugins/unittest_plugins.cpp

```diff
@@ -2971,7 +2971,8 @@ TEST (test_tensor_filter, reopen_tflite_02_p)
   ASSERT_TRUE (prop != NULL);
 
   prop->fwname = fw_name;
-  prop->model_file = test_model;
+  prop->model_files = (const gchar **) &test_model;
```

Comment:
string list 'model_files' should contain null ptr. (if 1 model file, {model1, null} should be assigned.)

Reply:
Thanks, will be fixed